### PR TITLE
Added neutrino library

### PR DESCRIPTION
**Pull request by ghost**
Created a neutrino library web project.
Added [prefherder](https://github.com/mozilla-frontend-infra/perf-goggles-demo/blob/master/src/utils/perfherder.js) file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "perf-goggles",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:csd713/perf-goggles.git",
+  "author": "Chiranjeevi <jeevi146@gmail.com>",
+  "license": "MIT",
```
Comment (armenzg) [inline package.json]: I checked perf-goggles and I don't know why I chose MIT.
Could you please remove this and instead add a LICENSE file like [this](https://github.com/mozilla/firefox-code-coverage-frontend/blob/master/LICENSE)?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "perf-goggles",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:csd713/perf-goggles.git",
+  "author": "Chiranjeevi <jeevi146@gmail.com>",
```
Comment (armenzg) [inline package.json]: Do you mind keeping my original authorship? As I will have to maintain it long-term.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "perf-goggles",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:csd713/perf-goggles.git",
```
Comment (armenzg) [inline package.json]: Use this https://github.com/mozilla-frontend-infra/perf-goggles.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1 @@
+export default () => 'Welcome to perf-goggles';
```
Comment (armenzg) [inline src/index.js]: Interesting. Is this added by default by the Neutrino wizard?

**Comment by armenzg** [comment]
This is looking good.
As the next steps will be to make the perf-goggles-demo use this package as a dependency and removing src/utils/perfherder.js from that repository.

After that it will be looking at making fx-health-dashboard use this package.

Now, are you happy to keep working on this? Or want me to look for another thing as well?

File: src/index.js (hunk 4 above)
Comment (ghost) [reply src/index.js]: Yes! Would you recommend removing this file?

**Comment by ghost** [comment]
I have made recommended changes. Please review @armenzg 

If there are only few more steps pending with this task then I'd like to finish this before moving on to next thing!

**Pull request closed without merging**